src/Arriba/Arriba.Server/Startup.cs

```diff
@@ -46,8 +48,10 @@ public void ConfigureServices(IServiceCollection services)
                 });
             });
 
-            services.AddOAuth(serverConfig);
+            var arribaManagementServiceFactory = new ArribaManagementServiceFactory(new DatabaseFactory().GetDatabase());
 
+            services.AddOAuth(serverConfig);
+            services.AddSingleton(arribaManagementServiceFactory.CreateArribaManagementService());
```

Comment:
This does not feel necessary. Seems like the existing host construction could be used here. I agree we'll probably want to pick on and not have mixed DI implementations, but for now it feels like adding this extra factory is not necessary